homeassistant/components/tuya/select.py

```diff
@@ -219,6 +219,30 @@
             entity_category=EntityCategory.CONFIG,
         ),
     ),
+    # Robot Vacuum
+    # https://developer.tuya.com/en/docs/iot/fsd?id=K9gf487ck1tlo
+    "sd": (
+        SelectEntityDescription(
```

Comment:
- Let's add an icon, to make it look nice.
- Needs a custom device class to provide translations for the different options

Reply:
Done! Please review.

Reply:
Add custom device class